### PR TITLE
fixes#2402-NTR-pelvic wall

### LINKED ISSUE #2402: NTR: pelvic wall
**Comment:** The following term is needed to indicate the origin of the tissues used establish cell lines annotated in Cellosaurus. Many thanks

**Preferred term label:** pelvic wall
**Synonyms** wall of pelvis
**Definition** The pelvic wall includes the urogenital and pelvic diaphragm. [https://link.springer.com/chapter/10.1007/978-3-642-40006-3_7]
**Parent term** 
anatomical wall UBERON:0000060 
pelvis UBERON:0002355
**Your nano-attribution (ORCID)** 0000-0002-0819-0473

### PR DESCRIPTION
**Pull request by paulacalipho**
fixes #2402-NTR-pelvic wall

### CODE REVIEW

**Review by bvarner-ebi — APPROVED** [review]
Consider revising text definition.

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224422,6 +224422,15 @@ is_a: UBERON:0006314 ! bodily fluid
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-12-13T12:30:22Z xsd:dateTime
 
+[Term]
+id: UBERON:8480043
+name: pelvic wall
+def: "The wall of the pelvis is formed by bones and ligaments that are lined with muscles covered with fascia and parietal peritoneum." [ISBN:1451110324]
```
[the remark this answers is not in the document]
Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:224428]: @bvarner-ebi I have removed the bones from the definition to model similar to UBERON:0003697 'abdominal wall'.
Here is the link to the book:
https://books.google.ch/books?id=vb4AcUL4CE0C&pg=PA242&redir_esc=y#v=onepage&q&f=false
image

[the remark this answers is not in the document]
Comment (bvarner-ebi) [reply src/ontology/uberon-edit.obo:224428]: @paulacalipho, I would include a reference to bones in the definition for 'pelvic wall'.

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -224422,6 +224422,15 @@ is_a: UBERON:0006314 ! bodily fluid
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-12-13T12:30:22Z xsd:dateTime
 
+[Term]
+id: UBERON:8480043
+name: pelvic wall
+def: "The tissues that surround the organs that are present within the pelvic region of trunk. The pelvic wall tissue is composed of muscles, fascia and parietal peritoneum." [ISBN:1451110324]
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:224428]:
```suggestion
def: "The tissues that surround the organs that are present within the pelvic region of the trunk. The pelvic wall tissue includes bone, muscle, fascia and parietal peritoneum." [ISBN:1451110324]
```

**Pull request merged**